bundles/http/Citrix/0b4d1a6b-30f3-4134-a354-42fa008b757e/f3b61af4-efd0-43c2-9a08-a467fc8242c1.js

```diff
@@ -0,0 +1,1294 @@
+let parser = library.load("xml2js")
+let jp = library.load("jsonpath-plus")
+const moment_lib = library.load("moment-timezone")
```

Comment:
The good habit is to use just `moment`
```suggestion
const moment = library.load("moment-timezone")
```

Reply:
Change Done